packages/effect/src/ManagedRuntime.ts

```diff
@@ -23,13 +44,29 @@ export declare namespace ManagedRuntime {
    * @since 3.4.0
    */
   export type Error<T extends ManagedRuntime<any, any>> = [T] extends [ManagedRuntime<infer _R, infer E>] ? E : never
+  /**
+   * @since 3.9.0
+   * @category models
+   */
+  export interface Variance<in out R, in out ER> {
```

Comment:
We don't really need a variance type as the generics are used by the members. You can just add `[TypeId]: TypeId` for now.